src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.html

```diff
@@ -29,6 +29,11 @@ <h1>Home</h1>
   </div>
 </div>
 
+<div *ngIf="isCopyingCourse" class="margin-top-bottom-30px">
```

Comment:
Let's try to use [Bootstrap styles](https://getbootstrap.com/docs/5.0/utilities/spacing/) here

Reply:
Thanks! I've now changed it to bootstrap